Automattic/jetpack
# Add post publish quick share buttons

Related to https://github.com/Automattic/jetpack-reach/issues/37

## Proposed changes:

* Add sharing buttons UI to be used in post-publish panel in another PR

### Other information:


## Jetpack product discussion
pdrWKz-1b4-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

* Add the buttons to some panel UI
* Check if they work alright

<img width="194" alt="Screenshot 2023-09-18 at 5 12 38 PM" src="https://github.com/Automattic/jetpack/assets/18226415/c63b35de-032f-4c24-964a-fd92b7640be1">
